src/components/SiteDetails.js

```diff
@@ -11,33 +11,37 @@ const checklistItems = [
  * Component to show details about Frogtoberfest in the homepage.
  */
 const SiteDetails = () => (
-  <div className="rounded mx-auto shadow w-3/4 sm:w-1/2 lg:w-1/3">
-    <div className="px-6 py-4">
-      <div className="font-bold mb-4">
-        <p className="text-lg mb-1">
-          Frogtoberfest, inspired from Hacktoberfest is a month-long open source contribution challenge for
-          Leapfroggers.
-        </p>
+  <div className="md:py-4 mb-6">
+    <div className="rounded mx-auto shadow w-3/4 sm:w-1/2">
+      <div className="px-6 py-4">
+        <div className="font-bold mb-4">
+          <p className="mb-4 leading-normal text-lg mb-1">
+            Frogtoberfest, inspired from Hacktoberfest is a month-long open source
+            contribution challenge for Leapfroggers.
+          </p>
+        </div>
+        <ul className="p-0">
+          {checklistItems.map((item, index) => (
+            <li className="flex leading-tight items-center mb-3" key={index}>
+              <div className="rounded-full fill-current text-green">
+                <svg
+                  className="w-6 h-6 align-middle"
+                  viewBox="0 0 24 24"
+                  fill="none"
+                  stroke="currentColor"
+                  strokeWidth="2"
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                >
+                  <path d="M22 11.08V12a10 10 0 1 1-5.93-9.14"></path>
+                  <polyline points="22 4 12 14.01 9 11.01"></polyline>
+                </svg>
+              </div>
+              <span className="text-gray-700 text-m ml-2">{item}</span>
```

Comment:
`text-gray-700` is from a newer version. We can remove this as it is unnecessary. @sagunji